Libraries/Image/Image.windows.js

```diff
@@ -113,6 +113,21 @@ var Image = React.createClass({
 
   statics: {
     resizeMode: ImageResizeMode,
+
+    getSize(
```

Comment:
The other missing features relate to: 
1) ShouldNotifyLoadEvents (**added**)
2) RCTTextInlineImage